app/src/main-process/menu/build-default-menu.ts

```diff
@@ -150,6 +150,12 @@ export function buildDefaultMenu(
         click: emit('show-branches'),
       },
       separator,
+      {
+        label: __DARWIN__ ? 'Go to Summary' : 'Go to &Summary',
+        id: 'go-to-summary',
+        accelerator: 'CmdOrCtrl+S',
```

Comment:
Feels a lot like save :/ but I cant think of anything better

Reply:
I knowww. 😕

Reply:
![image](https://user-images.githubusercontent.com/7467062/46543635-f49e7b80-c886-11e8-8b82-40d5ccb1e76b.png)

Maybe `CmdOrCtrl+3` would make more sense?

Reply:
In a way, you can think of committing as saving a change to the repo so `CmdOrCtrl+S` sort of makes sense since a commit message is required.

1. `CmdOrCtrl+S`
2. Write a commit message
3. `CmdOrCtrl+Enter`

Reply:
@desktop/maintainers care to weigh in?

Reply:
I agree it makes sense under View

Reply:
`CmdOrCtrl+G` Or `CmdOrCtrl+3` (1 and 2 also jump in the UI) work for me. We can always change it later. Just a touch worried if we add another “Go to X” menu item

Reply:
The commit summary element is within the Changes tab, so I'd rather leave future numbered shortcuts for top-level components for now.

Reply:
@shiftkey `CmdOrCtrl+G` works for me - I've updated the hotkey accordingly.